Blockzilla/SearchSuggestionsPromptView.swift

```diff
@@ -0,0 +1,124 @@
+        promptContainer.layer.cornerRadius = promptCornerRadius
+        addSubview(promptContainer)
+        
+        promptContainer.snp.makeConstraints{ make in
+            make.top.equalTo(self).offset(8)
+            make.bottom.equalTo(self).offset(-8)
+            make.leading.equalTo(self).offset(6)
+            make.trailing.equalTo(self).offset(-6)
+        }
+        
+        promptTitle.text = UIConstants.strings.searchSuggestionsPromptTitle
+        promptTitle.textColor = UIConstants.Photon.Grey10
+        promptTitle.font = UIFont.systemFont(ofSize: 18, weight: UIFont.Weight.bold)
+        promptTitle.textAlignment = NSTextAlignment.center
+        promptTitle.numberOfLines = 0
+        promptTitle.lineBreakMode = .byWordWrapping
+        promptContainer.addSubview(promptTitle)
+
+        promptTitle.snp.makeConstraints{ make in
+            make.top.equalTo(promptContainer).offset(20)
+            make.leading.equalTo(promptContainer).offset(10)
+            make.trailing.equalTo(promptContainer).offset(-10)
+        }
+        
+        promptMessage.text = String(format: UIConstants.strings.searchSuggestionsPromptMessage, AppInfo.productName)
+        promptMessage.textColor = UIConstants.Photon.Grey10
+        promptMessage.font = UIFont.systemFont(ofSize: 14)
+        promptMessage.textAlignment = NSTextAlignment.center
+        promptMessage.numberOfLines = 0
+        promptMessage.lineBreakMode = .byWordWrapping
+        promptContainer.addSubview(promptMessage)
+        
+        promptMessage.snp.makeConstraints { make in
+            make.top.equalTo(promptTitle.snp.bottom).offset(5)
+            make.leading.equalTo(promptContainer).offset(10)
+            make.trailing.equalTo(promptContainer).offset(-10)
+        }
+        
+        buttonBorderTop.backgroundColor = UIConstants.Photon.Grey10.withAlphaComponent(0.2)
+        addSubview(buttonBorderTop)
+        
+        buttonBorderTop.snp.makeConstraints { make in
+            make.top.equalTo(promptMessage.snp.bottom).offset(20)
+            make.leading.trailing.equalTo(promptContainer)
+            make.height.equalTo(0.5)
+        }
+        
+        buttonBorderMiddle.backgroundColor = UIConstants.Photon.Grey10.withAlphaComponent(0.2)
+        addSubview(buttonBorderMiddle)
+        
+        buttonBorderMiddle.snp.makeConstraints { make in
+            make.top.equalTo(buttonBorderTop.snp.bottom)
+            make.bottom.equalTo(promptContainer)
+            make.width.equalTo(0.5)
+            make.height.equalTo(40)
+            make.centerX.equalTo(self)
+        }
+
+        disableButton.accessibilityIdentifier = "SearchSuggestionsPromptView.disableButton"
+        disableButton.setTitle(UIConstants.strings.searchSuggestionsPromptDisable, for: .normal)
+        disableButton.titleLabel?.font = UIFont.systemFont(ofSize: 18)
+        disableButton.layer.cornerRadius = 8.0
```

Comment:
Again, UIConstant would be best for this, especially since we're repeating it in this file.